libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/TestOptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// Test options.
+    /// </summary>
+    public class TestOptions
```

Comment:
>TestOptions [](start = 17, length = 11)

Do you actually need this class?  It is not really a $kind in emitEvent.  For the custom event the payload is an arbitrary object, right?  And you can take that arbitrary object and put it into memory without a strongly typed payload.  The nice thing about this is that people can put in any values they like in the test options structure.